.devcontainer/devcontainer.json

```diff
@@ -3,9 +3,11 @@
     "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
     "features": {
         "ghcr.io/devcontainers/features/go:1": {
-            "version": "1.21"
+            "version": "1.23"
+        },
+        "ghcr.io/guiyomh/features/golangci-lint:0":{
+            "version": "1.60.1"
```

Comment:
@vhvb1989 Just a note, I locked these to a fixed version (which matches what we pulled in CI) instead of allowing it to float to latest.  This felt more in line with the spirit of things.